### DISCUSSION TITLE
How to manage workspace via bit

### DISCUSSION THREAD

**Discussion by MatthD** [teambit/bit#8812] [General]
Hello ! 

Regarding pnpm suggested to use Bit for monorepo deps management we are trying to figure out how to deal with current architecture of packages.

We have this 

---node
-----apps
-------api
-------otherapp
-----packages
--------core
--------attr

So our packages `can` be used inside our `apps`
We had a typescript config via `path` property, for eg we could load `@namespace/core`  from our `apps/api/sample.ts`

I figure out to do it via Bit with `bit add packages/core --namespace "@namespace"` but it seems the `@` cause issue, anyway, removing the `@` it seems every file become a symlink, but when I had a file into `packages/core` I need to `bit install` everytime again.
So the dev experience is worst than with typescript alias

How to deal with that ?

**Comment by itaymendel** [comment]
think about bit like any other monorepo tool. Essentially it creates and manages symlinks for each components in node-modules, so you don't need to set up anything. Bit already scopes components.

i am unsure about what you mentioned in regards to the bit-install item. you do not need to run this command after every time you create a component.
